config/crd/common/bases/services.k8s.aws_adoptedresources.yaml

```diff
@@ -40,12 +40,6 @@ spec:
                 description: AWSIdentifiers provide all unique ways to reference an
                   AWS resource.
                 properties:
-                  additionalKeys:
```

Comment:
something not right, this shouldn't have been removed?

Reply:
You're right its in my generated files don't know how it got removed